packages/xo-server-audit/src/index.js

```diff
@@ -20,38 +20,56 @@ const DEFAULT_BLOCKED_LIST = {
   'audit.checkIntegrity': true,
   'audit.generateFingerprint': true,
   'audit.getRecords': true,
+  'backup.list': true,
+  'backupNg.fetchFiles': true,
```

Comment:
Why?

Reply:
We don't want to log the reading actions.

Reply:
I think it makes sense to log explicit manual action, such as downloading files from backups.

Reply:
Done :+1: